### PR TITLE
CI: Add support for AppVeyor (Windows)

### PR DESCRIPTION
**Pull request by hhslepicka**
Included in this PR are:

- support for AppVeyor to run the automated tests against Windows.
- Fix for one of the tests that was *nix specific.
- Badges for Codecov and AppVeyor
- .coveragerc to control what is included on coverage report.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+build: false
+
+clone_folder: C:\projects\pydm
+
+platform:
+  #- x86
```
Comment (teddyrendahl) [inline appveyor.yml]: Remove comment?

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  - "conda build -q conda-recipe --python=%TRAVIS_PYTHON_VERSION% --output-folder bld-dir"
+  - "conda config --add channels \"file:///C:/projects/pydm/bld-dir\""
+  - "conda create -q -n test-environment python=%TRAVIS_PYTHON_VERSION% pydm --file dev-requirements.txt"
+  - activate test-environment
+
+test_script:
+  - python run_tests.py
+
+after_test:
+  - codecov
```
Comment (teddyrendahl) [inline appveyor.yml]: Do you want both `appveyor` and `Travis` to report code coverage? I might just pick one...

Comment (hhslepicka) [reply appveyor.yml]: Take a look here: https://github.com/codecov/support/wiki/Merging-Reports

Comment (teddyrendahl) [reply appveyor.yml]: Nice. Just wasn't sure that `codecov` differentiated the two

File: appveyor.yml (hunk 1 above)
Comment (hhslepicka) [reply appveyor.yml]: Done! Addressed by latest commit.

**Comment by danielballan** [comment]
Nice! The bar has been raised....

**Pull request merged**